bridges/testing/framework/js-helpers/wrapped-assets-balance.js

```diff
@@ -8,7 +8,7 @@ async function run(nodeName, networkInfo, args) {
     const bridgedNetworkName = args[2];
     while (true) {
         const foreignAssetAccount = await api.query.foreignAssets.account(
-            { parents: 2, interior: { X1: { GlobalConsensus: bridgedNetworkName } } },
+            { parents: 2, interior: { X1: [{ GlobalConsensus: bridgedNetworkName }] } },
```

Comment:
if this works, then I am wondering why other similar stuff works :D e.g.: https://github.com/paritytech/polkadot-sdk/blob/master/bridges/testing/environments/rococo-westend/bridges_rococo_westend.sh#L332
and https://github.com/paritytech/polkadot-sdk/blob/master/bridges/testing/framework/utils/generate_hex_encoded_call/index.js#L99

Reply:
I'm looking into it. Testing with PJS to see what's the issue

Reply:
Effectively it does not work 😅. I saw that when migrating to v4 and thought it would be a fix, but it doesn't work. Back to the drawing board

Reply:
This is definitely the way to do it. So I'll have to change those other spots as well.